toolchain/check/convert.cpp

```diff
@@ -134,7 +134,7 @@ static auto MakeElemAccessNode(Context& context, Parse::Node parse_node,
     // special case.
     auto index_id = block.AddNode(SemIR::IntegerLiteral(
         parse_node, context.GetBuiltinType(SemIR::BuiltinKind::IntegerType),
-        context.semantics_ir().AddInteger(llvm::APInt(32, i))));
+        context.semantics_ir().integers().Add(llvm::APInt(32, i))));
```

Comment:
I wonder whether it'd be a useful convenience to have `Context` also expose access to the value stores?

Reply:
Let's discuss this once the other things (functions, classes, etc) are also switched over (coming soon). I think it might be a little odd to have one and not the other, but also might be a fair amount to add.